bosh/secrets.example.yml

```diff
@@ -34,12 +33,40 @@ properties:
     component_name: cf-swarm-containers-broker
     docker_url: secret
 
-
 networks:
 - name: default
-  type: dynamic
+  type: manual
```

Comment:
This is preferable to using a `dynamic` `default` network, then making a separate one with `type:vip`?